src/core/public/legacy_platform/legacy_platform_service.ts

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
__question:__ there is no need to change anything in this PR, just wondering if we should put all legacy related public stuff into `public/legacy_compat` like we do for server and as mentioned in https://github.com/elastic/kibana/issues/12466#issuecomment-386809436 (mentioned only `server/legacy_compat` though)?